### PR TITLE
oled page initial

### PR DESCRIPTION
**Pull request by jlpoltrack**
Title, captured all of the screens and the data/options available on them.

### CODE REVIEW

**Comment by olliw42** [comment]
maybe useful, or not
mlrs-v10-oled-00
mlrs-v10-oled-01
mlrs-v10-oled-02
mlrs-v10-oled-03
mlrs-v10-oled-04
mlrs-v10-oled-05
mlrs-v10-oled-06
mlrs-v10-oled-07

**Comment by olliw42** [comment]
I could imagine that in each of the tables at the botton (or the top) a picture of the screen is shown (scaled down to something proper, not that large as in the previous post)

File: docs/OLED.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+mLRS allows for a 0.96", 128x64, SSD1306 OLED display to be connected to a Tx module using I2C.  When paired with a 5-way joystick (4 directions + select) this allows for one to update parameters and display link statistics.  The OLED is particularly useful on radios that do not have a 480 x 272 color LCD and cannot utilize the color Lua script.
+
+## Hardware Configuration
+
+Instructions for adding an OLED display to the FRM303 module [here](https://github.com/olliw42/mLRS-docu/blob/master/docs/FLYSKY_FRM303.md#oled-display-addition).
+
+## OLED Screens & Navigation
+
+From the main screen, you can navigate the through 4 main screens using the up and down directions, these contain the following information:
```
Comment (olliw42) [inline docs/OLED.md:13]: shouldn't it be "... navigate through ... " 
maybe also a '.' before "these conatin..."

Comment (jlpoltrack) [reply docs/OLED.md:13]: Good catch, updated.

**Comment by jlpoltrack** [comment]
Images added to the tables and updated some values as well.

**Comment by olliw42** [comment]
your last change corrects the muliple CliLineEnd
but the table xml still looks strange - at least at first sight -- it has a couple of empty <tr></tr>. I may be wrong of course.

**Comment by jlpoltrack** [comment]
Those are necessary for formatting - one tip is that you can do a preview by switching to this branch to see how Github will display the page (top left, dark mode doesn't screenshot well):

image

**Comment by olliw42** [comment]
> Those are necessary for formatting

ah, ok, thx for the clarification

yeah, I do this preview all time ... it was just the "code" which looked odd to me ... (some html parsers accept errors in the html, so the visual is not always telling the "truth")

but since it's all good:

lgtm !!

**Pull request merged**